docs/tutorials/add-catalog.md

```diff
@@ -29,35 +29,33 @@ This catalog is distributed as an image [quay.io/operatorhubio/catalog](https://
     apiVersion: olm.operatorframework.io/v1alpha1
     kind: ClusterCatalog
     metadata:
-      name: operatorhubio
+      name: <catalog_name>
     spec:
       source:
         type: Image
         image:
           ref: <catalog_image>
-          pollInterval: <poll_interval_duration>
+          pollIntervalMinutes: <poll_interval_duration>
     ```
 
     `catalog_name`
     :   Specifies the image reference for the catalog you want to install, such as `quay.io/operatorhubio/catalog:latest`.
 
     `poll_interval_duration`
-    :   Specifies the interval for polling the remote registry for newer image digests.
-            The default value is `24h`.
-            Valid units include seconds (`s`), minutes (`m`), and hours (`h`).
-            To disable polling, set a zero value, such as `0s`.
+    :   Specifies the number of minutes for polling the remote registry for newer image digests.
+            This field is optional. To disable polling, unset the field.
 
     ``` yaml title="Example `operatorhubio.yaml` CR"
     apiVersion: olm.operatorframework.io/v1alpha1
     kind: ClusterCatalog
     metadata:
-      name: operatorhub
+      name: operatorhubio
```

Comment:
👍 I did not find more places 
Only here to fix the name of the catalog

Reply:
Later in the doc there are references to another name. A bit below:

```
clustercatalog.olm.operatorframework.io/operatorhubio created
```